CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,151 @@
+
+> *"People from different backgrounds are likely to have different interpretations of what words or behaviors are considered acceptable. It's somewhat ironic that in an attempt to draw a more diverse developer base, code of conduct moderation, if improperly conducted, could potentially alienate users from other countries who don't have the same social conditioning as those in North America [or other Western cultures]."*[2]
+
+1. The Code of Conduct can be enforced through a range of passive and active means. We recommend minimal use for first offenders but allow rapid escalation if required (depending on the severity of the breach).
+2. Passive means may include a public warning against members of the community in general breaching the Code of Conduct. Where a moderator is involved in the conversation in question,  we strongly recommend against that moderator being the instigator of this warning if possible. Passive means may also involve reminders of good examples and bad examples of behaviour relating to the code of conduct.
+3. Active means may range from temporary communicative suppression (mutes), temporary bans, to permanent bans.
+4. Moderators must assemble evidence for punishments enacted should they be requested to provide it.
+   1. The moderator is not required to include their interpretation or deliberation of the material. This however may be raised by the aggrieved party (or a representative) in which case the decision will be reviewed within 7 days by another moderator who is independent of the first.
+
+<!-- If a member of the community or public is ever concerned with any actions of our moderation or code of conduct, they may let any of our moderators know. Should they feel this is inadequate or inappropriate for whatever reason, they may communicate with @shayanhabibi. Communications will be confidential unless told otherwise. -->
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+<!-- ## Revision
+
+Revision of this document would require a democratic decision involving the majority of active Nim-Works organisational members within 7 days of the revision proposed. This does not apply when the revision does not meaningfully change the content within (ie it is related to formatting) -->
+
+## References
+
+1.	Singh V, Bongiovanni B, Brandon W. Codes of conduct in Open Source Software—for warm and fuzzy feelings or equality in community? Software Quality Journal. 2021.
+
+2.	Li R, Pandurangan P, Frluckaj H, Dabbish L. Code of Conduct Conversations in Open Source Software Projects on Github. Proceedings of the ACM on Human-Computer Interaction. 2021;5(CSCW1):1-31.
+
+3.	Dunbar-Hester C. Hacking Technology, Hacking Communities: Codes of Conduct and Community Standards in Open Source. MIT Case Studies in Social and Ethical Responsibilities of Computing. 2021.
+
+4.	Tourani P, Adams B, Serebrenik A, editors. Code of conduct in open source projects 2017: IEEE.
```

Comment:
```suggestion
[^1]:	Singh V, Bongiovanni B, Brandon W. Codes of conduct in Open Source
      Software—for warm and fuzzy feelings or equality in community? Software
      Quality Journal. 2021.

[^2]:	Li R, Pandurangan P, Frluckaj H, Dabbish L. Code of Conduct Conversations
      in Open Source Software Projects on Github. Proceedings of the ACM on
      Human-Computer Interaction. 2021;5(CSCW1):1-31.

[^3]:	Dunbar-Hester C. Hacking Technology, Hacking Communities: Codes of
      Conduct and Community Standards in Open Source. MIT Case Studies in
      Social and Ethical Responsibilities of Computing. 2021.

[^4]:	Tourani P, Adams B, Serebrenik A, editors. Code of conduct in open
      source projects 2017: IEEE.
```

Github footnotes; then these can be referenced in the text above via `[^1]` etc.

So in the beginning when talking about evidence based research popping in the refs will create little superscripts links to the bottom here.